includes/onnxruntime/onnxruntime_cxx_api.h

```diff
@@ -599,6 +599,7 @@ struct CustomOpApi {
   OrtValue* KernelContext_GetOutput(OrtKernelContext* context, _In_ size_t index, _In_ const int64_t* dim_values, size_t dim_count);
 
   void ThrowOnError(OrtStatus* result);
+  const OrtApi& Api() { return api_; }
```

Comment:
These headers are directly copied for ORT release, which suppose not be changed.

Reply:
Retrieving input string tensors is not easy. The function used for the other types fail on Linux, there is no KernelContext_* methods to do so. The only way is go handle them directly from the API. I need to raise an issue on onnxruntime. This is a temporary solution.

Reply:
I removed it by using a trick which should be replaced by something cleaner when the C API is updated.